### PR TITLE
Example: sample

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

File: Tasks/sample.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Get one random element from array
+
+const SAMPLE = arr => {
+  arr = arr[
+    Math.floor(Math.random()* arr.length)
+  ]; return arr
+};
```
Comment (Selbonyka) [inline Tasks/sample.js:7]: This function could be broken up into two:
```
const pickRandomNumber = (length) => {
    return Math.floor(Math.random() * length);
};

const takeSample = (...args) => {
    return (args = args[pickRandomNumber(args.length)]);
};
```

File: Tasks/sample.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// Get one random element from array
+
+const SAMPLE = arr => {
+  arr = arr[
+    Math.floor(Math.random()* arr.length)
+  ]; return arr
```
Comment (Selbonyka) [inline Tasks/sample.js:6]: If the code won't be broken up into two functions for some reason, at least these lines should be optimised:
```
const SAMPLE = (arr) => {
  return (arr = arr[Math.floor(Math.random() * arr.length)]);
};
```

**Review by Selbonyka — CHANGES_REQUESTED** [review]
I have optimized this code, breaking it up into different functions, making them inline. As well now the code supports inputting individual numbers/words, so it doesn't have to take input as an array.


```js
// Get one random element from array

const pickRandomNumber = (length) => Math.floor(Math.random()*length);

const takeSample = (...args) => args[pickRandomNumber(args.length)];

const result = takeSample(123, 456, 789, 222, 333, 444);
console.log(result);

```

**Comment by Selbonyka** [comment]
In case the commit doesn't work, here's this code on python:

```
import random

trial_list = [123, 456, 789, 222, 333, 444]
sample = random.choice(trial_list)
print(sample)
```

File: Tasks/sample.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+// Get one random element from array
+
+const SAMPLE = arr => {
```
Comment (Selbonyka) [inline Tasks/sample.js:3]: camelCase should be used instead of CAPS

File: Tasks/sample.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+  arr = arr[
```
Comment (Selbonyka) [inline Tasks/sample.js:4]: The way it's formatted isn't optimal.

File: Tasks/sample.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+    Math.floor(Math.random()* arr.length)
```
Comment (Selbonyka) [inline Tasks/sample.js:5]: Since we aren't working with huge numbers, it's better to use ~~ instead of Math.floor, as it is faster

**Comment by tshemsedinov** [comment]
- Use strict mode
- You can't call function before declaration
- Prefer not to use ~~

**Pull request closed without merging**